<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>b2 PipeLine</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_projectInfo">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout_project">
        <property name="horizontalSpacing">
         <number>1</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_project">
          <property name="text">
           <string>Project:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_project">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_projectPath">
        <property name="horizontalSpacing">
         <number>1</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_projectPath">
          <property name="text">
           <string>Project Path:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_projectPath">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_user">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="horizontalSpacing">
         <number>1</number>
        </property>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_user">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_user">
          <property name="text">
           <string>User:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_main">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>700</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>700000</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_assetBroswer">
         <attribute name="title">
          <string>Asset Browser</string>
         </attribute>
         <widget class="QFrame" name="frame_assetDir">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>481</width>
            <height>351</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_assetType">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>101</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Asset Type</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_assetTypeDelete">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>290</y>
             <width>51</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_assetTypeNew">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>290</y>
             <width>61</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>New..</string>
           </property>
          </widget>
          <widget class="QListView" name="listView_assetType">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>111</width>
             <height>261</height>
            </rect>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_assetCompDelete">
           <property name="geometry">
            <rect>
             <x>420</x>
             <y>290</y>
             <width>51</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_assetComponent">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>10</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Component</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_assetNew">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>290</y>
             <width>171</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>New..</string>
           </property>
          </widget>
          <widget class="QListView" name="listView_assetComponent">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>30</y>
             <width>101</width>
             <height>261</height>
            </rect>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_assetCompNew">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>290</y>
             <width>51</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>New..</string>
           </property>
          </widget>
          <widget class="QListView" name="listView_asset">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>30</y>
             <width>231</width>
             <height>261</height>
            </rect>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_asset">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>10</y>
             <width>56</width>
             <height>12</height>
            </rect>
           </property>
           <property name="text">
            <string>Asset</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_assetDelete">
           <property name="geometry">
            <rect>
             <x>300</x>
             <y>290</y>
             <width>61</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </widget>
         <widget class="QFrame" name="frame_assetPreview">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>370</y>
            <width>481</width>
            <height>311</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_assetPreviewImage">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>320</width>
             <height>240</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>N:/b1Env/maya/2014/icons/noPreview.png</pixmap>
           </property>
          </widget>
         </widget>
         <zorder>frame_assetPreview</zorder>
         <zorder>frame_assetDir</zorder>
        </widget>
        <widget class="QWidget" name="tab_shotBrowser">
         <attribute name="title">
          <string>Shot Browser</string>
         </attribute>
         <widget class="QFrame" name="frame_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>370</y>
            <width>481</width>
            <height>311</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_shotPreviewImage">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>320</width>
             <height>240</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>N:/b1Env/maya/2014/icons/noPreview.png</pixmap>
           </property>
          </widget>
         </widget>
         <widget class="QFrame" name="frame_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>481</width>
            <height>351</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_14">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>101</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Sequence</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_14">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>290</y>
             <width>51</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_15">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>290</y>
             <width>51</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>New..</string>
           </property>
          </widget>
          <widget class="QListView" name="listView_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>101</width>
             <height>261</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_16">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>290</y>
             <width>51</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_15">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>10</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Component</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_17">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>290</y>
             <width>81</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>New..</string>
           </property>
          </widget>
          <widget class="QListView" name="listView_8">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>30</y>
             <width>101</width>
             <height>261</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_18">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>290</y>
             <width>51</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>New..</string>
           </property>
          </widget>
          <widget class="QListView" name="listView_9">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>30</y>
             <width>131</width>
             <height>261</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_16">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>10</y>
             <width>56</width>
             <height>12</height>
            </rect>
           </property>
           <property name="text">
            <string>Shot</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_19">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>290</y>
             <width>51</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_20">
           <property name="geometry">
            <rect>
             <x>420</x>
             <y>290</y>
             <width>51</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
          <widget class="QListView" name="listView_10">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>30</y>
             <width>101</width>
             <height>261</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_21">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>290</y>
             <width>51</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>New..</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_17">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>10</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Layer</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_history">
        <item>
         <widget class="QLabel" name="label_history">
          <property name="text">
           <string>History</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_History">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>690</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="verticalLayout_historyAdd">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>348</width>
             <height>688</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_historySub"/>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_fileLocation">
      <item>
       <widget class="QLabel" name="label_fileLocation">
        <property name="text">
         <string>File Location</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_fileLocation"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_fileLocation">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>&amp;Setup</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuSetup"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
